core/templates/dev/head/pages/exploration-editor-page/improvements-tab/templates/answer-details-modal.template.html

```diff
@@ -18,7 +18,7 @@ <h3 class="modal-title">
       <tr ng-repeat="learnerAnswerInfo in getLearnerAnswerInfos()" class="oppia-feedback-tab-row">
         <th scope="row">
           <input type="checkbox" ng-change="selectLearnerAnswerInfo(learnerAnswerInfo)"
-                 ng-model="myVal">
+                 ng-model="myVal" ng-disabled="!isUserLoggedIn">
```

Comment:
The checkboxes should not be shown at all (rather than shown but disabled) if the user is not logged in.